test_Car.py

```diff
@@ -0,0 +1,23 @@
+from unittest import TestCase
+
+from car.Car import Car
+
+
+class TestCar(TestCase):
+    def setUp(self):
+        self.car = Car()
+
+
+class TestBrake(TestCar):
+    def test_brake(self):
+        self.car.brake()
+
+        self.assertEqual(self.car.speed, 0)
+
+    def test_accelerate_brake(self):
+        self.car.accelerate()
+
+        self.assertEqual(self.car.speed, 5)
+
```

Comment:
The car should brake after you accelerate, then assert that the speed is 0. This shows that the brakes work, when the car is moving.